packages/sdk/src/peer_manager/peer_manager.ts

```diff
@@ -34,20 +34,26 @@ type GetPeersParams = {
 };
 
 export enum PeerManagerEventNames {
-  Connect = "filter:connect",
-  Disconnect = "filter:disconnect"
+  FilterConnect = "filter:connect",
+  FilterDisconnect = "filter:disconnect",
+  StoreConnect = "store:connect"
 }
 
-interface IPeerManagerEvents {
+export interface IPeerManagerEvents {
   /**
    * Notifies about Filter peer being connected.
    */
-  [PeerManagerEventNames.Connect]: CustomEvent<PeerId>;
+  [PeerManagerEventNames.FilterConnect]: CustomEvent<PeerId>;
 
   /**
    * Notifies about Filter peer being disconnected.
    */
-  [PeerManagerEventNames.Disconnect]: CustomEvent<PeerId>;
+  [PeerManagerEventNames.FilterDisconnect]: CustomEvent<PeerId>;
+
+  /**
+   * Notifies about a Store peer being connected.
+   */
+  [PeerManagerEventNames.StoreConnect]: CustomEvent<PeerId>;
```

Comment:
this new event doesn't seem to be used, is it for future PRs? 

also, don't you need `StoreDisconnect` event too?

Reply:
> this new event doesn't seem to be used, is it for future PRs?

Yes, see `QueryOnConnect`

> also, don't you need StoreDisconnect event too?

nah